### PR TITLE
PS demo library

### PR DESCRIPTION
**Pull request by nancymic2**
note, moving content to partials 

Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

Create PS demo library page

- https://pr-6287-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/demos/
- create partials for each entry
-

### CODE REVIEW

File: site/en/docs/privacy-sandbox/demos/index.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+subhead: >
+  Demos and colabs walk you through the Privacy Sandbox APIs.
+description: >
+  Demos and colabs walk you through the Privacy Sandbox APIs.
+date: 2023-05-29
+authors:
+  - alexandrawhite
+---
+
+There are a number of demos available for the Privacy Sandbox APIs. Most of them will require you to be familiar with the APIs, so if you haven't read the docs, check out the links we've listed along with the demos here.
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/demos/index.md:13]:
```suggestion
There are a number of demos available for the Privacy Sandbox APIs. Most of them will require you to be familiar with the APIs, so if you haven't read the docs, check out the links we've listed along with the demos.
```

File: site/en/docs/privacy-sandbox/demos/index.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+## Demos and colabs
+
+{% Details %}
+{% DetailsSummary %}
+### Protected Audience API (FLEDGE)
+{% endDetailsSummary %}
+
+{% Columns %}
+{% Column %}
+This demo uses FLEDGE to join ad interest groups on two advertiser sites,
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/demos/index.md:28]:
```suggestion
The demo uses FLEDGE to join ad interest groups on two advertiser sites,
```

File: site/en/docs/privacy-sandbox/demos/index.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+and then initiates an on-device auction to select an ad for display on a publisher site.
+
+[Demo](https://fledge-demo.glitch.me/)
+
+[Docs](/docs/privacy-sandbox/fledge-api/)
+
+{% endColumn %}
+{% Column %}
+
+<a href="https://fledge-demo.glitch.me/">{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/Tejkps8XvgWoobfGLMVO.png", alt="ALT_TEXT_HERE", width="484", height="418" %}</a>
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/demos/index.md:38]: Alt text missing

File: site/en/docs/privacy-sandbox/demos/index.md [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+{% DetailsSummary %}
+### Protected Audience API (FLEDGE)
+{% endDetailsSummary %}
+
+{% Columns %}
+{% Column %}
+This demo uses FLEDGE to join ad interest groups on two advertiser sites,
+and then initiates an on-device auction to select an ad for display on a publisher site.
+
+[Demo](https://fledge-demo.glitch.me/)
```
Comment (kevinkiklee) [inline site/en/docs/privacy-sandbox/demos/index.md:31]: https://protected-audience-demo.web.app/ is now the new main demo for PAAPI

File: site/en/docs/privacy-sandbox/demos/index.md (hunk 3 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/demos/index.md]: Thanks @anusmitaray. fixed

File: site/en/docs/privacy-sandbox/demos/index.md (hunk 4 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/demos/index.md]: Oh thanks!

File: site/en/_partials/privacy-sandbox/demos/shared-storage.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+[Demo](https://shared-storage-demo.web.app/)
+
+[Docs](/docs/privacy-sandbox/attribution-reporting/shared-storage/)
+
+{% endColumn %}
+{% Column %}
+
+<a href="https://shared-storage-demo.web.app/">{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/dOFtwOaI8ldFfwgmG5El.png", alt="Shared Storage API 
+ demo.", width="751", height="521" %}</a>
```
Comment (anusmitaray) [inline site/en/_partials/privacy-sandbox/demos/shared-storage.md:18]: Is this space (between API and demo) a typo?

**Review by anusmitaray — APPROVED** [review]

File: site/en/_partials/privacy-sandbox/demos/shared-storage.md (hunk 5 above)
Comment (nancymic2) [reply site/en/_partials/privacy-sandbox/demos/shared-storage.md:18]: oh yes!

**Review by kevinkiklee — APPROVED** [review]

**Pull request merged**